src/components/ProgressBar/styles.scss

```diff
@@ -43,7 +44,7 @@
     background: $color-success;
 }
 
-.rainbow-progress-bar_circular .rainbow-progress-bar__value {
+.rainbow-progress-bar .rainbow-progress-bar__value {
     border-radius: 0.5rem;
```

Comment:
.rainbow-progress-bar already have the ```border-radius: 0.5rem;``` and remove the:
```
.rainbow-progress-bar_circular {
     border-radius: 0.5rem;
 }
```
above in the code since circular will not be more a variant
and remove all vertical class since we will not use them.